lago/workdir.py

```diff
@@ -503,3 +502,16 @@ def set_current(prefix_name, parent_workdir, **kwargs):
         workdir(str): path to the workdir to change the current of
     """
     parent_workdir.set_current(new_current=prefix_name)
+
+
+class LagoPrefixAlreadyExistsError(utils.LagoException):
+    def __init__(self, name, path):
+        super().__init__(
+            dedent(
+                """
+                Failed Prefix name {} already exists in workdir {}
```

Comment:
I think that we should keep the old error message.
This error message is wrong because creating a prefix isn't relate to running a VM.
Running a VM occurs in `lago start`, creating a prefix occurs in `lago init`.